CONTRIBUTING.md

```diff
@@ -1,27 +1,32 @@
 # Contributing to `std::simd`
 
 Simple version:
-1. Fork it and git clone it
-2. Create your feature branch: git checkout -b my-branch
+1. Fork it and `git clone` it
+2. Create your feature branch: `git checkout -b my-branch`
 3. Write your changes.
-4. Test it: cargo test. Remember to use appropriate features.
-5. Commit your changes: git commit add ./path/to/changes && git commit -m 'Fix some bug'
-6. Push to the branch: git push origin my-branch
+4. Test it: `cargo test --features `. Remember to use appropriate SIMD features.
```

Comment:
Do we think we'll ever have any cargo features?

Reply:
...No, that was an error!

Reply:
I had meant to say "set RUSTFLAGS", which it now does.